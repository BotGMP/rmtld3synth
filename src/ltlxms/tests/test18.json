[ "SubSetEq", [ "Proposition", "a" ], [ "Proposition", "b" ] ]