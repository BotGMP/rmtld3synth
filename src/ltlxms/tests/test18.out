(forall ((xs Real) (ys Real)) (=> a b))

